CONTRIBUTE.md

```diff
@@ -7,49 +7,87 @@
 ### Contribution of OEO content
 Please read [OEO wiki](https://github.com/OpenEnergyPlatform/ontology/wiki/Best-Practice-Principles) carefully.
 
-### Philosophy
+### Workflow
 
 The development of a feature for this repository is inspired from the workflow described 
 by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
 
-1. Create [an issue](https://help.github.com/en/articles/creating-an-issue) on the github repository to describe the addition to the ontology. Discussion about the implementation details should occur within this issue 
+1. Create [an issue](https://help.github.com/en/articles/creating-an-issue) on the github repository
+
+    Choose the right issue for you among the choices you have on github:
+
+    A) Including a new term to the ontology. Only 1 new term (with a few subclasses, if at all)
+    
+    B) Restructuring existing parts of the ontology
+    
+    C) Other issue
+
+    Discussion about the implementation details should occur within this issue. [Assign a project](https://help.github.com/en/github/managing-your-work-on-github/adding-issues-and-pull-requests-to-a-project-board#adding-issues-and-pull-requests-to-a-project-board-from-the-sidebar) (default "Issues") to the issue
+
 2. Create a branch from dev to work on your issue (see below, the "Branch name convention" section)
     ```bash
     git checkout -b feature/myfeature dev
     ```
-    As all need to edit the same `oeo.omn` file, it is better to merge one additions as fast as possible to avoid merging conflicts. 
-    Therefore, it is best to create a pull request and merge it at the end of each work session if the ontology changed much.
-    This is different from a more conventional workflow where each issue is linked to only one pull request
+    As all need to edit the same `oeo.omn` file, it is better to merge one changes as fast *as possible* (i.e. do not wait for 6 months) to avoid merging conflicts
 3. Open [protégé](https://protege.stanford.edu/) and work on the ontology
-    Please refrain from editing the `oeo.omn` file in a text editor. 
-    If you do so, open protégé and check that the nodes which you modified do not have any problems
-4. Describe briefly what you changed in the `CHANGELOG.md` file and commit the file last
+
+    One can also edit the `oeo.omn` file in a text editor
```

Comment:
Above you wrote that one should refrain from editing `oeo.omn` in a text editor

Reply:
What I understood from the hackathon discussion was that it wa s ok to make modification in text editor as long as the file was finally saved using protege, is it right @MGlauer ?

Reply:
That is also my understanding. However this last step to open and save the `oeo.omn` should explicitly stated in the CONTRIBUTE.md

Reply:
@l-emele - you can see the changes made in the file on github by clicking on the "File changes" Tab. This will show you [this page](https://github.com/OpenEnergyPlatform/ontology/pull/98/files)

The last step you mention was already mentioned under point 4. of the CONTRIBUTE.md :)